pkgs/by-name/ex/exegol/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  pkgs,
+  fetchPypi,
+}:
+pkgs.python311Packages.buildPythonPackage rec {
+  pname = "Exegol";
+  version = "4.2.5";
+  format = "setuptools";
+  doCheck = false;
+
+  propagatedBuildInputs = with pkgs.python311.pkgs; [
+    pyyaml
+    gitpython
+    docker
+    requests
+    rich
+    argcomplete
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-X/3Bu0LI/O7sSI5Y3ot497ArGfcRoBPbxkdT+mieapM=";
+  };
+
+  meta = with lib; {
+    description = "Fully featured and community-driven hacking environment";
+    longDescription = ''
+      Exegol is a community-driven hacking environment, powerful and yet
+      simple enough to be used by anyone in day to day engagements. Exegol is
+      the best solution to deploy powerful hacking environments securely,
+      easily, professionally. Exegol fits pentesters, CTF players, bug bounty
+      hunters, researchers, beginners and advanced users, defenders, from
+      stylish macOS users and corporate Windows pros to UNIX-like power users.
+    '';
+    homepage = "https://github.com/ThePorgs/Exegol";
+    changelog = "https://github.com/ThePorgs/Exegol/releases/tag/${version}";
+    license = licenses.gpl3;
```

Comment:
gpl3 is imprecise, should be either `gpl3Only` or `gpl3Plus`

Reply:
Alright thank you ! I could not find the difference between gpl3Only and gpl3Plus in documentation but comparing with other projects I think that gpl3Only is the right one.

Reply:
The difference is whether the project is licensed under the GPLv3 or the GPLv3 + any later version. Can't be guessed by looking at other packages.

Reply:
Alright, I asked Exegol maintainers, we're going for gpl3Only.